minizip/crypt.h was removed from the packet minizip-1.2.11_2. 

with "#define NOUNCRYPT" libkml can be built without "crypt.h"
https://github.com/void-linux/void-packages/commit/952ac913cf
https://github.com/madler/zlib/pull/229

--- a/src/kml/base/contrib/minizip/unzip.c	2015-12-21 18:23:05.000000000 +0100
+++ b/src/kml/base/contrib/minizip/unzip.c	2020-11-26 15:45:55.033835816 +0100
@@ -42,7 +42,7 @@
 #include <zlib.h> //RR
 #include "unzip.h"
 #include "iomem_simple.h"
-#undef NOUNCRYPT
+#define NOUNCRYPT
 
 #ifdef STDC
 #  include <stddef.h>
